jujugui/static/gui/src/app/views/utils.js

```diff
@@ -1513,15 +1513,15 @@ YUI.add('juju-view-utils', function(Y) {
         } else {
           charm = charm[0];
           var config = {},
-              options = charm.get('options');
+              options = charm.get ? charm.get('options') : charm.options;
```

Comment:
I believe any data out of jujulib that the gui gets is converted to a model; if it isn't that should probably be marked as a bug.